torch/csrc/distributed/autograd/utils.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/distributed/autograd/functions/sendrpc_backwards.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+// This method is used to attach the 'send' autograd function to the autograd
+// graph when we use RPC. This method creates a new 'send' autograd function
+// and attaches the provided IValues as next_edges to the 'send' function. Only
+// IValues of type 'Tensor' are attached.
+// TODO: Support Tensors which might be nested inside other IValue types
+//       (ex: List, Tuple)
```

Comment:
This is actually a serious limitation, because it means that operations which take tensor lists like `torch.stack` silently don't work with the current implementation right now. You don't have to make it work immediately, but we should really give a good error message if you try to use something bad.